lms/static/scripts/frontend_apps/utils/google-picker-client.js

```diff
@@ -98,19 +140,7 @@ export class GooglePickerClient {
    */
   async showPicker() {
     const pickerLib = await this._gapiPicker;
-    let accessToken;
-    try {
-      accessToken = await this._authorizeDriveAccess();
-    } catch (err) {
-      if (err.error === 'popup_closed_by_user') {
-        throw new PickerCanceledError();
-      } else if (err.error) {
-        // Error returned by the Google API client (not an instance of `Error`)
-        throw new Error(err.error);
-      } else {
-        throw err;
-      }
-    }
+    const accessToken = await this._authorizeDriveAccess();
```

Comment:
This code is simplified because `_authorizeDriveAccess` should now always throw an error, and handles mapping the "popup closed" message to a `PickerCanceledError` itself.